moveit_simple/src/moveit_simple.cpp

```diff
@@ -770,6 +770,21 @@ void Robot::updateState(const sensor_msgs::JointStateConstPtr& msg)
   robot_state_->setVariablePositions(msg->name, msg->position);
 }
 
+std::vector<double> Robot::getCurrentRobotState(void) const
+{
+  std::lock_guard<std::recursive_mutex> guard(m_);
+  ros::spinOnce();
+  std::vector<double> current_joint_positions;
+  current_robot_state_->update();
+  current_robot_state_->copyJointGroupPositions(joint_group_->getName(), current_joint_positions);
+  return current_joint_positions;
+}
+
+
+void Robot::updateCurrentState(const sensor_msgs::JointStateConstPtr& msg)
```

Comment:
@shaun-edwards! You are referring to not using the ROS messages in [updateState](https://github.com/shaun-edwards/moveit_simple/pull/21/files#diff-8ae580aedd7e6266898c159ae2e4a380R767), right?